packages/react-ui-ag/src/ToggleDropDown/ToggleDropDown.js

```diff
@@ -0,0 +1,103 @@
+import React, { createElement } from 'react'
+import { Motion, spring } from 'react-motion'
+import themed from 'react-themed'
+import PropTypes from 'prop-types'
+import { ToggleButton } from '@rentpath/react-ui-core'
+import classnames from 'classnames'
+import autobind from 'autobind-decorator'
+import { parseArgs } from '@rentpath/react-ui-utils'
+
+@themed(/^ToggleDropDown/, {
+  pure: true,
+})
+
+export default class ToggleDropDown extends React.Component {
+
+  static propTypes = {
+    className: PropTypes.string,
+    theme: PropTypes.object,
+    onClick: PropTypes.func,
+    height: PropTypes.number,
+    children: PropTypes.node,
+    openButtonContents: PropTypes.node,
+    closeButtonContents: PropTypes.node,
+    movementDamping: PropTypes.number,
+    movementStiffness: PropTypes.number,
+    visible: PropTypes.bool,
+  }
+
+  static defaultProps = {
+    theme: {},
+    height: 220,
+    movementDamping: null,
+    movementStiffness: null,
+    visible: true,
+  }
+
+  constructor(props) {
+    super(props)
+    this.state = { visible: props.visible }
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (!(this.props.visible === nextProps.visible)) {
+      this.setState({
+        visible: nextProps.visible,
+      })
+    }
+  }
+
+  @autobind
+  handleToggle(value) {
+    this.setState({ visible: value })
+    if (this.props.onClick) this.props.onClick(value)
+  }
+
+  render() {
+    const {
+      theme,
+      height,
+      visible,
+      className,
+      children,
+      movementDamping,
+      movementStiffness,
+    } = this.props
+
+    return (
+      <div
+        className={classnames(theme.ToggleDropDown, className)}
+        data-tid="toggleDropDown"
+      >
+        <Motion style={{ x: spring(this.state.visible ? 0 : height, movementStiffness, movementDamping) }}>
+          {({ x }) =>
+            (
+              <div>
+                <ToggleButton
+                  className={classnames(
+                    theme.ToggleDropDown_ToggleButton,
```

Comment:
I don't think we need both of these since I'm assuming the `ToggleButton` component already has a class by this name.